include/drivers/mem_manage/system_mem_manage.h

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright (c) 2021 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Memory Management Driver APIs
```

Comment:
How about putting this in `include/drivers/mm/` and calling it `system_mm.h` instead?